db/migrate/20240723130645_rename_research_step_to_research_activity.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+class RenameResearchStepToResearchActivity < ActiveRecord::Migration[7.1]
```

Comment:
If we don't push this on production, you can remove this migration, because the last productional state is Vocabulary::MarketplaceLocation :)